tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -157,7 +157,7 @@ $KeepScssVar: val;
 
   .Keep;
   .Keep();
-  .Keep(4px) !important;
+  .Keep(4px)!important;
```

Comment:
What happened here?

Reply:
I don't really know. The file is autogenerated from jest and my guess is that its a change in the alpha release.  I'll investigate a bit more

Reply:
So it turns out I had the wrong version of jest cached in `node_modules`.  Fixed just by manually removing the package and rerunning yarn.